audiofile/core/convert.py

```diff
@@ -33,12 +35,18 @@ def convert_to_wav(
         duration: return only a specified duration in seconds
         offset: start reading at offset in seconds
 
+    Raises:
+        RuntimeError: if ``file`` is broken or not a supproted format
```

Comment:
```suggestion
        RuntimeError: if ``file`` is broken or not a supported format
```

Reply:
There is no way to find out whether it's a broken file or not a supported format, right?

Reply:
No, I don't think so.